runtime.md

```diff
@@ -34,22 +34,23 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is started to when it is stopped or crashes unexpectedly.
```

Comment:
On Tue, Nov 17, 2015 at 04:21:21PM -0800, Vish Kannan wrote:
> > -This event needs to be captured by runc to run onstop event handlers.
> > +The lifecycle describes the timeline of events that happen from when a container is started to when it is stopped or crashes unexpectedly.
> 
> nit: How about `container is created` instead of `started`?

Both of Vish's suggestions sound like improvements to me.